spaceship.zsh

```diff
@@ -642,6 +642,8 @@ spaceship_ruby() {
     ruby_version=$(chruby | sed -n -e 's/ \* //p')
   elif _exists rbenv; then
     ruby_version=$(rbenv version-name)
+  elif _exists ruby; then
```

Comment:
The problem here is that we are trying not to show the `system` versions. With this change, it will show ruby version for every ruby-specific folder, even when we have the system version.